sample/mpp-library/src/commonMain/kotlin/com/icerockdev/library/LoginViewModel.kt

```diff
@@ -11,6 +11,7 @@ import dev.icerock.moko.fields.core.validations.ValidationResult
 import dev.icerock.moko.fields.core.validations.matchRegex
 import dev.icerock.moko.fields.core.validations.minLength
 import dev.icerock.moko.fields.core.validations.notBlank
+import dev.icerock.moko.fields.livedata.BaseLiveDataFormField
```

Comment:
why here this type?

Reply:
So sorry, My mistake in tests performance livedata and fixes after core changes. Removed